### DISCUSSION TITLE
Something went wrong! file an issue at

### DISCUSSION THREAD

**Discussion by vishesharya** [anuraghazra/github-readme-stats#2592] [General]
Something went wrong! file an issue at https://tiny.one/readme-stats
Maximum retries exceeded Please add an env variable called PAT_1 with your github token in vercel
https://github.com/vishesharya in this readmi file

**Comment by rickstaa** [comment]
Duplicate of #1471.

**Comment by Myloveken** [comment]
Something went wrong, file and issue at a readme profile: please add an env variable called pat_1 with your github token in vercel. help me please!
